runtime/themes/dark_plus.toml

```diff
@@ -35,8 +35,8 @@
 
 "string" = { fg = "#ce9178" }
 "string.regexp" = { fg = "regex" }
-"number" = { fg = "#b5cea8" }
-"escape" = { fg = "#d7ba7d" }
+"constant.numeric" = { fg = "#b5cea8" }
+"constant.character" = { fg = "#d7ba7d" }
```

Comment:
`constant.character.escape`?

Reply:
I believe that they both get styled the same, or at least they used to be.